imblearn/base.py

```diff
@@ -66,24 +68,39 @@ def fit_resample(self, X, y):
         y : array-like, shape (n_samples,)
             Corresponding label for each sample in X.
 
+        sample_weight : array-like, shape (n_samples,) or None
+            Sample weights.
+
+
         Returns
         -------
-        X_resampled : {array-like, sparse matrix}, shape \
+        X_resampled : {ndarray, sparse matrix}, shape \
 (n_samples_new, n_features)
             The array containing the resampled data.
 
-        y_resampled : array-like, shape (n_samples_new,)
+        y_resampled : ndarray, shape (n_samples_new,)
             The corresponding label of `X_resampled`.
 
+        sample_weight_resampled : ndarray, shape (n_samples_new,)
+            Resampled sample weights. This output is returned only if
+            ``sample_weight`` was not ``None``.
+
+        idx_resampled : ndarray, shape (n_samples_new,)
```

Comment:
Could you explain why this should be returned from `fit_resample`, rather than stored as an attribute?